src/pages/cli/migration/cli-migrate-aws-account.mdx

```diff
@@ -0,0 +1,14 @@
+export const meta = {
+  title: `Migrate project to another AWS account`,
+  description: `Amplify CLI can be used to easily re-create resources in a different AWS account.`
+};
+
+Amplify CLI can be used to easily re-create resources in a different AWS account.
+
+Run the following steps to migrate an exisiting project to another AWS account.
```

Comment:
@flozender - Can you please correct the typo on this line - change 'exisiting' to '**existing**'?

Reply:
Done, thank you!